tests/Feature/QuestionApiTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class QuestionApiTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    /**
+     * A basic feature test example.
+     */
+    public function test_add_new_question_success(): void
+    {
+        $response = $this->postJson('/api/questions', [
+            'question' => 'Will this request work?',
+            'hint' => "Let's see",
+            'points' => 2,
+            'quiz_id' => 2,
+        ]);
+        $response->assertValid(['message']);
```

Comment:
You should also do an assertStatus(201) along with an `$this->assertDatabaseHas()` to make sure the data actually got saved.